api/src/scripts/updateDoi.ts

```diff
@@ -209,7 +209,12 @@ const updateDoi = async (): Promise<void> => {
     let index = 1;
 
     for (const publication of simplifiedPublications) {
-        await helpers.updateDOI(publication.doi, publication, publication.References).catch((err) => console.log(err));
+        const abstractedReferences = publication.References.map((reference) => {
+            const { publicationVersionId, ...referenceRest } = reference;
+
+            return { publicationId: publication.id, ...referenceRest };
```

Comment:
The "publicationId" is not used by the _updateDOI_ helper. There's no need to map the references here